docs/reference/connectors/bigquery/configuration.mdx

```diff
@@ -0,0 +1,294 @@
+
+```json
+{
+  "connection_uri": {
+    "value": "...",
+    "variable": "..."
+  },
+  "connection_pool_settings": {
+    "max_connections": 10,
+    "min_idle": 1,
+    "connection_timeout": 30000,
+    "initialization_fail_timeout": 30000
+  },
+  "schemas": ["public"],
+  "tables": [...],
+  "functions": [],
+  "native_operations": {}
+}
+```
+
+### Property: connection_uri
+
+The connection URI for the BigQuery database. This is a required field that can be specified either as a direct string
+value or as a reference to an environment variable:
+
+```json
+{
+  "connection_uri": {
+    "value": "jdbc:bigquery://https://www.googleapis.com/bigquery/v2:443;ProjectId=your-project-id;OAuthType=0;OAuthServiceAcctEmail=your-service-account-email;OAuthPvtKeyPath=/path/to/key.json;"
+  }
+}
+```
+
+Or using an environment variable:
+
+```json
+{
+  "connection_uri": {
+    "variable": "BIGQUERY_JDBC_URL"
+  }
+}
+```
+
+### Property: schemas
+
+This is an optional array of schema names to include in the introspection process. If not provided, all schemas will be
+included. **Any schema passed in the JDBC URL will take precedence.**
+
+Example:
+
+```json
+{
+  "schemas": ["schema1", "schema2"]
+}
+```
+
+### Property: tables
+
+An array of table definitions generated automatically during introspection. Each table definition includes metadata
+about the table structure, columns, primary keys, and foreign keys.
+
+Example:
+
+```json
+{
+  "tables": [
+    {
+      "name": "mydataset.customers",
+      "description": "Customer information table",
+      "category": "TABLE",
+      "columns": [
+        {
+          "name": "customer_id",
+          "description": "Unique customer identifier",
+          "type": {
+            "scalar_type": "INT64"
+          },
+          "nullable": false,
+          "auto_increment": false,
+          "is_primarykey": true
+        },
+        {
+          "name": "name",
+          "description": "Customer name",
+          "type": {
+            "scalar_type": "STRING"
+          },
+          "nullable": false,
+          "auto_increment": false
+        },
+        {
+          "name": "location",
+          "description": "Geographic location",
+          "type": {
+            "scalar_type": "GEOGRAPHY"
+          },
+          "nullable": true,
+          "auto_increment": false
+        },
+        {
+          "name": "tags",
+          "description": "Customer tags",
+          "type": {
+            "array_type": {
+              "scalar_type": "STRING"
+            }
+          },
+          "nullable": true,
+          "auto_increment": false
+        }
+      ],
+      "primary_keys": ["customer_id"],
+      "foreign_keys": {
+        "fk_customer_order": {
+          "column_mapping": {
+            "customer_id": "customer_id"
+          },
+          "foreign_collection": "mydataset.orders"
+        }
+      }
+    }
+  ]
+}
+```
+
+### Property: functions
```

Comment:
We don't support functions or native operations in bigquery right now. Do we want these sections at all?